### PR TITLE
refactor: replace initials

### PR DESCRIPTION
**Pull request by timosville**
Card: https://trello.com/c/SMMtdGyR/1677-pc-find-and-replace-v2

### CODE REVIEW

File: src/admin/config.yml [hunk 1]
```diff
@@ -1050,7 +1050,7 @@ collections:
                         name: "helpText",
                         widget: "text",
                         i18n: true,
-                        default: "Sign up to receive each day's Bible in One Year reading in your inbox",
+                        default: "Sign up to receive each day's the Bible with Nicky and Pippa Gumbel reading in your inbox",
```
Comment (paurakhsharma) [inline src/admin/config.yml:1053]:
```suggestion
                        default: "Sign up to receive each day's The Bible with Nicky and Pippa Gumbel reading in your inbox",
```

File: src/admin/index.njk [hunk 2]
```diff
@@ -4,13 +4,13 @@ eleventyExcludeFromCollections: true
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
-    <title>BiOY Marketing | Content Manager</title>
+    <title>BWBPG Marketing | Content Manager</title>
```
Comment (paurakhsharma) [inline src/admin/index.njk:10]:
```suggestion
    <title>BNAP Marketing | Content Manager</title>
```

File: src/faqs/ticket.njk [hunk 3]
```diff
@@ -6,7 +6,7 @@ pagination:
   alias: locale
 permalink: "{{ locale }}/faqs/ticket/index.html"
 activePage: faqs
-title: "Ask a question about BiOY"
+title: "Ask a question about BWBPG"
```
Comment (paurakhsharma) [inline src/faqs/ticket.njk:9]:
```suggestion
title: "Ask a question about BNAP"
```

**Review by paurakhsharma — CHANGES_REQUESTED** [review]
Looks great, please update these changed.

**Comment by timosville** [comment]
> Looks great, please update these changed.

Thanks. Done.

**Review by paurakhsharma — APPROVED** [review]
🚀

**Pull request merged**